### ISSUE TITLE
[MobRecon] Details about pre-training

### ISSUE THREAD

**Issue by vra** [SeanChenxy/HandMesh#37]
Hi @SeanChenxy ,

I am adopting the MobRecon network to train my custom model.  As a basic step, I try to train MobRecon without using pre-trained `densestack.pth`, however the validation pampjpe is higher. I searched the paper but not find too much details about pre-training. Could you give more details about this? e.g., is pre-training a task of 2d keypoints estimation? which loss is used, which datasets are used during it? Thanks :heart:

**Comment by SeanChenxy** [comment]
Hi, we brief pre-training in supplementary material. This task go towards 2D pose position and heatmap based on L1 loss. We use our synthesized data for pre-training (Due to commercial reason, I cannot release all our pre-training data). Please see Table 1 for PAMPJPE based on from-sctatch training.

**Comment by vra** [comment]
Thanks for you information, I will check the supplementary material.
